c2corg_ui/static/affiz.html

```diff
@@ -0,0 +1,14 @@
+<html>
+<head>
+<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
+<meta http-equiv="content-language" content="fr-FR" />
+<meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7" />
+<title></title>
+</head>
+<body leftmargin='0' topmargin='0' marginwidth='0' marginheight='0' style="background-color:transparent;">
```

Comment:
Right. I will make the change. Actually this file is a simple port of the v5 one: https://github.com/c2corg/camptocamp.org/blob/master/web/affiz.html